File: svcenter-front/src/app/models/naturelogPo.ts
import {PageBean} from "./pageBean";

/**
 * 服务器性能记录表
 *
 * @author zhangshengze
 */
export class NaturelogPo extends PageBean {

    // 服务器性能记录
    public id: number;
    // 性能详情
    public detail: string;
    // 服务器id
    public serverid: number;
    // 记录时间
    public addtime: string;

    // 用作页面展示 't_' 开头
    public t_id: number;
    public t_detail: string;
    public t_serverid: number;
    public t_addtime: string;

}

File: svcenter/src/main/java/com/croakzh/webfront/po/NaturelogPo.java
package com.croakzh.webfront.po;

import com.croakzh.webfront.po.base.PageBean;
import lombok.Data;

import java.io.Serializable;

/**
 * 服务器性能记录表
 *
 * @author croakzh
 * @since 2018-12-05
 */
@Data
public class NaturelogPo extends PageBean implements Serializable {

    /**
     * serialVersionUID 持久化ID
     */
    private static final Long serialVersionUID = 1L;

    /**
     * 服务器性能记录
     */
    private Integer id;

    /**
     * 性能详情
     */
    private String detail;

    /**
     * 服务器id
     */
    private Integer serverid;

    /**
     * 记录时间
     */
    private String addtime;

}
